spec/02-integration/13-vaults/05-ttl_spec.lua

```diff
@@ -221,5 +222,204 @@ describe("vault ttl and rotation (#" .. strategy .. ") #" .. vault.name, functio
   end)
 end)
 
+
+describe("#hybrid mode dp vault ttl and rotation (#" .. strategy .. ") #" .. vault.name, function()
+  local client
+  local admin_client
+  local secret = "my-secret"
+  local certificate
+
+  local tls_fixtures = {
+    http_mock = {
+      upstream_tls = [[
+        server {
+            server_name example.com;
+            listen 16799 ssl;
+
+            ssl_certificate         ../spec/fixtures/mtls_certs/example.com.crt;
+            ssl_certificate_key     ../spec/fixtures/mtls_certs/example.com.key;
+            ssl_client_certificate  ../spec/fixtures/mtls_certs/ca.crt;
+            ssl_verify_client      on;
+            ssl_verify_depth       3;
+            ssl_session_tickets    off;
+            ssl_session_cache      off;
+            keepalive_requests     0;
+
+            location = / {
+                echo 'it works';
+            }
+        }
+      ]]
+    },
+  }
+
+  tls_fixtures.dns_mock = helpers.dns_mock.new({mocks_only = true})
+  tls_fixtures.dns_mock:A {
+    name = "example.com",
+    address = "127.0.0.1",
+  }
+
+  local vault_fixtures = vault:fixtures()
+  vault_fixtures.dns_mock = tls_fixtures.dns_mock
+
+  lazy_setup(function()
+    helpers.setenv("KONG_LUA_PATH_OVERRIDE", LUA_PATH)
+    helpers.setenv("KONG_VAULT_ROTATION_INTERVAL", "1")
+
+    helpers.test_conf.loaded_plugins = {
```

Comment:
This assignment statement will cause a [flakiness](https://konghq.atlassian.net/browse/KAG-4240). Please just remove it.

Reply:
Fixed, @Water-Melon please review again